Sprint-1/errors/3.js

```diff
@@ -7,3 +7,8 @@ const last4Digits = cardNumber.slice(-4);
 // Then run the code and see what error it gives.
 // Consider: Why does it give this error? Is this what I predicted? If not, what's different?
 // Then try updating the expression last4Digits is assigned to, in order to get the correct value
+
+/* The slice function works with arrays and cardNumber was initially just one
+long number so while the TypeError given during debugging is that cardNumber.slice is not a function,
+it is actually that there is no array for the slice to work. By adding commas to 
+separate the integers in the const array and then running the code, it works */
```

Comment:
Thank you for all your comments and questions CJ, I am reviewing and committing the updates as I go. Your guidance is invaluable, I have been using ChatGPT to assist me when I get stuck but I am also realising that it would provide contextual answers based on my specific prompts, which may not always be generally accepted (or something like that!)